test/perf-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+require('babel-register');
+const Benchmark = require('benchmark');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+
+const filename = path.resolve(__dirname, 'screenshotter/ss_data.yaml');
+const data = yaml.load(fs.readFileSync(filename, 'utf-8'));
+
+console.log('compiling katex...');
+const katex = require('../katex');
+console.log('');
+
+const suite = new Benchmark.Suite;
+
+const testsToRun = [
+    "AccentsText",
+    "ArrayMode",
+    "GroupMacros",
+    "MathBb",
+    "SqrtRoot",
+    "StretchyAccent",
+    "Units",
+];
+
+for (const key of testsToRun) {
+    const value = data[key];
+    if (typeof value === "string") {
+        suite.add(key, () => katex.renderToString(value));
+    } else {
+        if (value.macros) {
+            const options = {
+                macros: value.macros,
+            };
+            suite.add(key, () => katex.renderToString(value.tex, options));
+        } else {
+            suite.add(key, () => katex.renderToString(value.tex));
+        }
+    }
+}
+
+suite.on('cycle', function(event) {
+    console.log(String(event.target));
+});
+
+suite.on('complete', function() {
+    console.log('Fastest is ' + this.filter('fastest').map('name'));
+});
+
+suite.run({
+    async: true,
```

Comment:
What does async do? None of the tests are async, so is this needed? (I guess I'm wondering if this affects the stability of timing or anything).

Reply:
I copied it from the benchmark.js example.  I removed and the tests still pass.  It shouldn't make a difference either way b/c all of our code is synchronous.